Impedrezine does something.

Kurgis opened the pull request:
Fixes #18595

Triples the brain loss from Impedrezine, should be sufficient to kill a person in large doses.
<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

Kurgis committed:
Impedrezine does something.

Should fix issue #18595

Kurgis committed:
Create Kurgis-PR-Impedrezine-Changes

Kurgis committed:
Rename Kurgis-PR-Impedrezine-Changes to Kurgis-PR-Impedrezine-Changes.yml

comma commented:
I think it'll still do jack shit on its own because it can't outrun brain regen (~1 damage per tick).
Your buff brought it from 0.02 damage per tick to 0.06, since 'removed' in that formula is 0.2u processed per tick.
IMO you either need to add some sort of chemical effect (a-la CE_TOXIC) that prevents brain regen, or raise it even more and pray.

Kurgis commented:
Well, that would certainly explain it. I don't want to add CE_TOXIC because then people would be able to mix it in with other toxins to make theirs more deadly. I'm just going to buff the damage and pray.

Kurgis committed:
Impedrezine BrainLoss increased

Increases Impedrezine brain loss from .2 to 5.25, dealing .05 damage per tick after brain regen.

PurpleMartinJCK commented:
Adding CE TOXIN won't make it combo with others afaik. It stops natural regen and a couple other bits and pieces.

comma commented:
Got conflicts

Kurgis committed:
Merge branch 'dev' into Impedrezine-Change
